app/Actions/Like/GetLikedPlaceAction.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Hedonist\Actions\Like;
+
+use Hedonist\Exceptions\Place\PlaceNotFoundException;
+use Hedonist\Repositories\Like\LikeRepository;
+use Hedonist\Repositories\Dislike\DislikeRepository;
+use Hedonist\Repositories\Place\PlaceRepository;
+use Illuminate\Support\Facades\Auth;
+
+class GetLikedPlaceAction
+{
+    private $likeRepository;
+    private $dislikeRepository;
+    private $placeRepository;
+
+    public function __construct(
+        LikeRepository $likeRepository,
+        DislikeRepository $dislikeRepository,
+        PlaceRepository $placeRepository
+    ) {
+        $this->likeRepository = $likeRepository;
+        $this->dislikeRepository = $dislikeRepository;
+        $this->placeRepository = $placeRepository;
+    }
+
+    public function execute(GetLikedPlaceRequest $request): GetLikedPlaceResponse
+    {
+        $liked = 'NONE';
```

Comment:
Create `Entities\Like\LikeStatus` class with static methods `none()`, `liked()`, `disliked()` and return these constants there.